docs/internals/controls-package.md

```diff
@@ -172,7 +432,17 @@ There are two main approaches that one can use:
 
 The following implementation records the step response of the plant and uses the relay method to estimate the ultimate gain (_Ku_) and oscillation period (_Tu_) of the plant.
 
-Several methods to calculate Kp, Ki and Kd are implemented.
+Several methods to calculate Kp, Ki and Kd are implemented:
+
+- ziegerNicholsPI
```

Comment:
well these seem like very random names. Do we want to explain any of this further?

Reply:
they are tuning different tuning methods for a PID loop. I don't think we need to add more info as it is very technical and not really viam related. maybe a note that you can google them if you want more info but idk if that's necessary

Reply:
Ok let's leave as is